Configurations.md

```diff
@@ -2392,6 +2392,70 @@ By default this option is set as a percentage of [`max_width`](#max_width) provi
 
 See also [`max_width`](#max_width) and [`use_small_heuristics`](#use_small_heuristics)
 
+## `single_line_let_else_max_width`
+
+Maximum line length for single line let-else statements.
+See the [let-else statement section of the Rust Style Guide](https://github.com/rust-lang/rust/blob/master/src/doc/style-guide/src/statements.md#else-blocks-let-else-statements) for more details on when a let-else statement may be written on a single line.
+A value of `0` (zero) means the divergent `else block` will always be formatted over multiple lines.
+Note this occurs when `use_small_heuristics` is set to `Off`.
+
+By default this option is set as a percentage of [`max_width`](#max_width) provided by [`use_small_heuristics`](#use_small_heuristics), but a value set directly for `single_line_let_else_max_width` will take precedence.
+
+- **Default value**: `50`
+- **Possible values**: any positive integer that is less than or equal to the value specified for [`max_width`](#max_width)
+- **Stable**: Yes
+
+#### `50` (default):
+
+```rust
+fn main() {
+    let Some(x) = opt else { return };
+
+    let Some(x) = some_very_very_very_very_long_name else {
+        return;
+    };
+
+    let Some(y) = opt else {
+        return;
+    };
+}
+```
+
+#### `0`:
+
+```rust
+fn main() {
+    let Some(x) = opt else {
+        return;
+    };
+
+    let Some(x) = some_very_very_very_very_long_name else {
+        return;
```

Comment:
Same comment wrt the semi

```suggestion
        return
```

Reply:
Unfortunately I can't remove the `;`. The `configuration_snippet_tests` would fail without it since rustfmt will add the `;` by default.